spec/rspec/mocks/space_spec.rb

```diff
@@ -235,17 +235,16 @@ def create_generations
     it 'raises ArgumentError with message if object is symbol' do
       space1 = Space.new
       object = :subject
-      expected_message = "Cannot proxy frozen objects. Symbols such as #{object} cannot be mocked or stubbed."
-
-      expect { space1.proxy_for(object) }.to raise_error(ArgumentError, expected_message)
+      expected_message = "Static objects such as #{object.class}:#{object} cannot be mocked or stubbed."
+      expect { space1.proxy_for(object).add_message_expectation(:foo) }.to raise_error(ArgumentError, expected_message)
     end
 
     it 'raises ArgumentError with message if object is frozen' do
       space1 = Space.new
-      object = "subject".freeze
+      object = {}.freeze
```

Comment:
Why?

Reply:
It's because frozen string acts like symbol (throws a type error), I'll revert the test and add it to the symbol check